boards/riscv/esp32c3_luatos_core/revision.cmake

```diff
@@ -0,0 +1,8 @@
+if(NOT DEFINED BOARD_REVISION)
```

Comment:
Thank you! I added the license identifier, but I'll keep this open in case @sylvioalves weighs in and says we should switch to using `board_check_revision()` instead of the custom logic.